schema.graphql

```diff
@@ -798,3 +798,76 @@ type TokenHourData @entity {
   # close price USD
   close: BigDecimal!
 }
+
+type Cellar @entity {
+  # Cellar Ethereum Contract Address
+  id: ID!
+
+  # Target Pool Address
+  pool: Pool!
+  feeTier: BigInt!
+
+  # Performance fee
+  managementFee: BigInt!
+
+  # NFLPs
+  nflps: [NFLP!]! @derivedFrom(field: "cellar")
+
+  # Tokens
+  token0: Token! # address
+  token1: Token! # address
+
+  # Lifetime Deposits
+  totalDepositAmount0: BigDecimal!
+  totalDepositAmount1: BigDecimal!
+  totalDepositUSD: BigDecimal!
```

Comment:
Might be good to have complenetary `withdraw` fields, even if no immediate use for them. Don't want to have to resync later if we end up needing them